bac_index.qmd

```diff
@@ -8,17 +8,22 @@ toc-depth: 4
 
 ### Tables
 
--   [CMT -- Cox Proportional Hazards Model Tables](./tables/cmt01.qmd)
```

Comment:
let's keep the highest level (e.g. `CMT`) and then below have indented on second level the `CMT1` etc.

Reply:
@danielinteractive please review the proposal for the bac_index.qmd, as commented in the commit, the high level links to the first sub template in that category, because there is no specific page (eg. CMT links to CMT1)